### DISCUSSION TITLE
2022.10.17.(월)

### DISCUSSION THREAD

**Discussion by kimziou77** [Rchicken/CheckMoi-v2#9] [회의록]
## 프로젝트 회의
주제: 책모이 v2
기간: 1~2달
회의: 주3회 (월, 수, 금)

## 회의 내용
- 인프라 관리
  - GCP 사용
  - IaC - 테라폼

- 리팩토링
  - 코드통일성 위주

- 추가요구사항구현
  - 채팅/ 알림/ 검색(ElasticSearch) 등 도입
  - 모니터링 구축 : 프로메테우스 & 그라파나 / pinpoint
- 테스트 관리
  - jacoco/sonarcube 제외
  - API 통합테스트 + 추가적인 유닛테스트
- 프로젝트 관리
  - 깃헙프로젝트
- 브랜치 관리
  -  github flow : main ← feat (squash merge)

## 다음 회의 내용
- 기본 CI/CD 설정
- 인프라 IAM설정
- 코드컨벤션 의견 정해오기
- 스웨거 알아오기
- PR/코드리뷰 방식
- 응답객체관리
- 사용해 볼 기술스택 학습

**Comment by zxcv9203** [comment]
회의록 작성하느라 고생하셨습니다~ 👍 

브랜치 전략에 대해 github flow라고 명시하는 건 어떠신가요?

로그 관리 부분은 -> 모니터링 구축으로 변경하는게 나을 것 같습니다.

**Reply by kimziou77** [reply]
수정했슴당 😺
